components/HelplineCard.jsx

```diff
@@ -38,14 +38,14 @@ const HelplineCard = ({
         <div className="flex space-x-7 items-start">
           {phone1 && (
             <a
-              className="font-mono text-gray-800 hover:text-gray-900 text-xl font-bold"
+              className="font-mono text-gray-800 hover:text-gray-900 text-l font-bold"
```

Comment:
It would be text-lg not text-l

Here is the documention for tailwind classes https://tailwindcss.com/docs/font-size#class-reference

Reply:
sorry about that. updated.